app/components/my-map/show-all-observations/template.hbs

```diff
@@ -1,46 +1,45 @@
-<GMap
-  class='main-map'
-  @lat='49.668437'
-  @lng='19.189728'
-  @zoom={{12}}
-  @styles={{this.myMapStyle}}
-  @minZoom={{10}}
-  @panControl={{false}}
-  @streetViewControl={{false}}
-  @onceOnIdle={{this.saveStartBounds}}
-  @onBoundsChanged={{this.onBoundsChange}}
-  as |map|
->
-
-  {{#each @observations as |observation|}}
-    <map.marker
-      @lat={{observation.latLocation}}
-      @lng={{observation.lngLocation}}
-      @draggable={{true}}
-      @onClick={{fn this.markerTooltipOpen observation}}
-      as |marker|
-    >
-      <marker.infoWindow @isOpen={{eq observation.id this.isOpenObservationId}}>
-        <div class='text-center'>
-          <div>
-            {{observation.birdname}}
-          </div>
-        </div>
-      </marker.infoWindow>
-    </map.marker>
-  {{/each}}
-</GMap>
-
-<div class='observations-under-map'>
-  {{#if this.isLoaded}}
-    <MyMap::ShowAllObservations::ObservationsOnMapTable
-      @observations={{@observations}}
-      @currentLatBounds={{this.currentLatBounds}}
-      @currentLngBounds={{this.currentLngBounds}}
-    />
-  {{else}}
-    <div class='small-spinner'>
-      <span class='loader'></span>
-    </div>
-  {{/if}}
+<GMap
+  class='main-map'
+  @lat='49.668437'
+  @lng='19.189728'

```

Comment:
How about create getters for that and use `navigator` to return current user coordinates? If will disallow access to geolocation in app then return these values. Take a look here https://whatwebcando.today/geolocation.html overall take a look on all features in https://whatwebcando.today 🚀